src/Apps/ArtAdvisor/07-Curated-Discovery/Components/Result/MarketingCollectionsRail.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, SkeletonBox } from "@artsy/palette"
+import { State } from "Apps/ArtAdvisor/07-Curated-Discovery/App"
+import { MarketingCollection } from "Apps/ArtAdvisor/07-Curated-Discovery/Components/Result/MarketingCollection"
+import { Rail } from "Components/Rail/Rail"
+import { FC, useEffect, useState } from "react"
+
+interface MarketingCollectionsRailProps {
+  state: State
+}
+
+export interface DiscoveryMarketingCollections {
+  id: string
+  imageUrl: string
+  slug: string
+  title: string
+}
+
+export const MarketingCollectionsRail: FC<MarketingCollectionsRailProps> = ({
+  state,
+}) => {
+  const [marketingCollections, setMarketingCollections] = useState<
+    DiscoveryMarketingCollections[]
+  >([])
+  const [isLoading, setIsLoading] = useState<boolean>(true)
+
+  useEffect(() => {
+    const params = new URLSearchParams()
+    params.append("concepts", state.goal) //TODO: improve use of goal
```

Comment:
In this first iteration, we're just adding the goal to the list of concepts to use in the `nearText` search. I will experiments with potentially better options, as this doesn't seem ideal (e.g. reranking based on `goal`, which will require a infra update to enable to module, unless we manually git a reranker).